docs/UserGuide.md

```diff
@@ -232,6 +324,16 @@ Format: `sort [sna/] [snd/] [ssa/] [ssd/] [so/]…​`
 
 ---
 
+### Clear all shows
+
+**Description:** Wanting to reset your current show list? Clear all shows in Trackermon's show list!
+
+Format: `clear`
+
+**Example & Output:** `clear`
+
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
```

Comment:
I feel that clearing doesnt need a image.

Reply:
Noted.